lib/marc/xml_parsers.rb

```diff
@@ -156,6 +172,7 @@ def attributes_to_hash(attributes)
   # which will cascade down to REXML if nothing better is found.
   #  
   module REXMLReader
+    include GenericPullParser
```

Comment:
What does `include GenericPullParser` do here, and how is it related to the new rescue behavior?

Reply:
`new_field` is defined in `MARC::GenericPullParser`, so we are including that as an interface here.